_sass/_aiml.scss

```diff
@@ -63,4 +67,107 @@
     padding: 3em;
     width: calc(100% - 6em);
     position: relative;
+    flex-shrink: 0.35;
+}
+.page-container{
+    display: flex;
+    justify-content: flex-start;
+}
+
+.current-dot{
+    height:10px;
+    width:10px;
+    background-color: $ai-blue;
+    border-radius: 50%;
+    display: inline-block;
+    box-shadow: 1px 1px 3px grey;
+}
+
+#desktop-sidebar{
+    width:200px;
+    margin:48px 10px;
+    margin-left: calc(50vw - 758px);
+    padding:10px 10px;
+    padding-bottom: 0px;
+    height: calc(100vh - 186px);
+    flex-grow: 0;
+    flex-shrink: 0;
+    border: solid;
+    border-radius: 10px;
+    overflow-y: auto;
+    overflow-x: hidden;
+    position: -webkit-sticky;
+    position: sticky;
+    top: 80px;
+    border-color: $ai-blue;
+    z-index: 1;
+    display:inline-block;
+    @media (min-height: 864px){
+        height: 680px;
+    }
+    @media (max-width: $desktop-width) {
+        display: none;
+    }
+    @media (max-width: 1516px){
+        margin-left: 10px;
+    }
+}
+#desktop-sidebar .sidebar-item{
+    padding-bottom: 15px;
+}
+#mobile-sidebar{
+    position: fixed;
+    top: 70px;
+    left: 16px;
+    box-sizing: border-box;
```

Comment:
On `master` I've already applied this!

Reply:
Not entirely sure what this is referring to, but I don't think I have any duplicate code anywhere. Was this resolved when we merged master into this branch?

Reply:
I've applied 

```css
* {
  box-sizing: border-box;
}
```

in `master`!

Reply:
Ok, I removed it!